firefox-ios/README.md

```diff
@@ -23,11 +23,11 @@ See readme at the root of the project for [the guidelines](https://github.com/mo
     brew install node
     pip3 install virtualenv
     ```
-1. Clone the repository:
+1. Clone the repository into your current directory:
     ```shell
     git clone https://github.com/mozilla-mobile/firefox-ios
     ```
-1. Install Node.js dependencies, build user scripts and update content blocker:
+1. Change directories to firefox-ios and install Node.js dependencies, build user scripts and update content blocker:
     ```shell
     cd firefox-ios
```

Comment:
I think this line is the only line that needs to change. bootstrap.sh is at the root folder, this makes it seem like it's in the firefox-ios subfolder.

Reply:
Hey Orla, I see what you mean. My only concern is that removing `cd firefox-ios`  might confuse people in a different way -- in particular they may not realize that they should change directories into the recently cloned repo (following the last step) before running boostrap.sh. I'm more than happy to go forward with removing it if it would be best though!

Another option is we could add a dedicated step for cding into the project root.


The change would look something like 
```md
1. Clone the repository:
    ```shell
    git clone https://github.com/mozilla-mobile/firefox-ios
    ```
1. Change directories to the project root
    ```shell
    cd firefox-ios
    ```
1. From the project root, install Node.js dependencies, build user scripts and update content blocker:
    ```shell
    sh ./bootstrap.sh
    ```
```
let me know your thoughts!